build.gradle

```diff
@@ -90,6 +90,9 @@ dependencies {
 
     // docker
     implementation "com.github.docker-java:docker-java:${dockerJavaVersion}"
+
+    // slf4j-test
+    testImplementation 'uk.org.lidalia:slf4j-test:1.2.0'
```

Comment:
In every test on `getIexecUploadToken`:
https://github.com/iExecBlockchainComputing/iexec-worker/blob/05fe1758a7156b17d78fdebfa581bdf3e189e644/src/test/java/com/iexec/worker/result/ResultServiceTests.java#L477

Reply:
Thank you :)
I like this answer, https://softwareengineering.stackexchange.com/a/426030, what do you think?

Reply:
I mainly agree with that answer. However, the reason I test these logs is because I want to be sure of the `if` branches the code goes into. I may try another approach if you prefer not to test logs.

Reply:
Note: I totally see why you do this!
Maybe we could cut in half and just something like `AssertContains("domain name")`, so we keep being quite free against log updates?

Reply:
I don't really like the `AssertContains` option as it could fit to different cases if strings are not detailed enough. And if they are detailed enough... Well, those are the entire log lines :grin:

I've tried a new option in https://github.com/iExecBlockchainComputing/iexec-worker/pull/453/commits/4f5b18574d6ce59dc5185cfb74396606222ad3cc, let me know if it seems OK for you!

Reply:
Yes, thank you for these tests and your patience :)